universe/marathon.json.mustache

```diff
@@ -104,20 +104,40 @@
         "image": "{{resource.assets.container.docker.spark_docker}}",
     {{/service.docker-image}}
     "network": "HOST",
-    {{#service.user}}
+    {{#service.docker_user}}
         "parameters": [
-        {
-        "key": "user",
-        "value": "{{service.user}}"
-        }
+            {
+                "key": "user",
+                "value": "{{service.docker_user}}"
+            }
+        ],
+    {{/service.docker_user}}
+    {{^service.docker_user}}
+        "parameters": [
+            {
+                "key": "user",
+                "value": "{{service.user}}"
```

Comment:
Question: Is `service.user` required? If not, will these parameters work as expected if it is not supplied? I'm not saying we should make the logic here more complex, I'm just asking.

Reply:
service user comes here from `config.json` and if not specified is set to default value of `nobody`.  one of the interesting findings was that if docker user parameter not supplied at all - then container is run as root even if Marathon service user is different.

Here is why `service.user` is alway present at the moment of template evaluation:
```
Variables in the mustache template will be evaluated from a union object created by merging three objects in the following order:
- Defaults specified in config.json
- User supplied options from either the DC/OS CLI or the DC/OS UI
- The contents of resource.json
```